service/runtime/handler.go

```diff
@@ -125,12 +129,34 @@ func (h *Handler) excludeHeader(headerName string) bool {
 	return false
 }
 
+func (h *Handler) getSignerConfig(clientID string) (SignerConfig, error) {
+	signerCfg, ok := h.signerConfigs[clientID]
+	if !ok {
+		return SignerConfig{}, errors.New("unknown clientID in header, please, check your signing proxy configuration")
+	}
+	return signerCfg, nil
+}
+
 func (h *Handler) ServeHTTP(rw http.ResponseWriter, inReq *http.Request) {
 	h.log.Log("\nSend request:\n")
 	ctx, cancel := context.WithTimeout(inReq.Context(), h.cfg.DefaultTimeout)
 	defer cancel()
 
-	url := fmt.Sprintf("%s%s", h.cfg.BaseUrl, inReq.URL.Path)
+	clientIDString := inReq.Header.Get(upvestClientID)
+	clientID, err := uuid.Parse(clientIDString)
```

Comment:
No upvest-client-id header in /auth/token request.

Reply:
But all the idea of this task is to have keys mapped by the `clientID`, so without clientID I can't choose which key to use, and I don't think that it's a good idea to parse request content to get ID from request's body.

Reply:
The idea was ( and is) even with the old config the new proxy must work properly.
Because  for the /auth/proxy request upvest-cient-id header is optional, better to not force users to add it.